src/System Application/App/Performance Profiler/src/ScheduledPerfProfilerImpl.Codeunit.al

```diff
@@ -220,4 +231,5 @@ codeunit 1932 "Scheduled Perf. Profiler Impl."
         ProfileCannotBeInThePastErr: Label 'A schedule cannot be set to run in the past.';
         ScheduleDurationCannotExceedRetentionPeriodErr: Label 'The performance profile schedule duration cannot exceed the retention period.';
         ScheduleEndTimeCannotBeEmptyErr: Label 'The performance profile schedule must have an end time.';
+        CannotCreateSchedulesForOtherUsersErr: Label 'Only user admins can create schedules for other users.';
```

Comment:
Sounds weird. `Only admin users...` maybe? Check with the design team, Jacob, as they should review labels.

Reply:
Fixed